runhouse/servers/cluster_servlet.py

```diff
@@ -71,6 +72,9 @@ async def aget_cluster_config(self) -> Dict[str, Any]:
         return self.cluster_config
 
     async def aset_cluster_config(self, cluster_config: Dict[str, Any]):
+        if "has_cuda" not in cluster_config.keys():
```

Comment:
is there any risk that a stale `has_cuda` value would be stored here? my point being any harm in just removing this check and setting it each time?

Reply:
hmmm, checking each time might increase the runtime a bit, but don't think it will cause massive latency. However, everytime the cluster is up (meaning the clusterServlet actor is up) => the "has cuda" value remains the same, therefore I don't see a real reason to set it each time.  Don't have strong opinion abouth this though. 

@jlewitt1 wdyt?

Reply:
gotcha yea so then we can keep as is then